docs/guides/setup/NCI/basics.rst

```diff
@@ -66,20 +82,34 @@ Set :guilabel:`Modules` to ``dea``.
    </video>
 
 
-Virtual Desktops with ARE
-=========================
+.. dropdown:: Troubleshooting: fe_sendauth: no password supplied
 
+   When the ``dea`` module is first run, a Datacube database role and ``.pgpass`` password
+   file is automatically created for you in your home directory. If you used a previous
+   version of the NCI's Virtual Desktop software (e.g. VDI, OOD), you may need to copy
+   this original ``.pgpass`` file into your new ARE home directory. If you cannot locate
+   your ``.pgpass`` file, please contact earth.observation@ga.gov.au to request your DEA
+   database account be reset.
 
 
 Setting up Digital Earth Australia
-----------------------------------
+==================================
+
+You will need to install DEA the first time you launch a session.
 
-In the terminal window run the command::
+In a terminal window on either JupyterLab or the ARE Virtual Desktop, run the command::
 
    sh /g/data/v10/public/digitalearthau/install.sh
 
 This will download the latest version of the `Digital Earth Australia notebooks
 repository <https://github.com/GeoscienceAustralia/dea-notebooks/tree/stable>`_
 into your home directory (e.g. :file:`~/dea-notebooks`).
 
+.. note:: DEA Notebooks is a large repository that will take up a large proportion of
```

Comment:
Does the `install.sh` file do anything besides clone dea-notebooks? If not, I'd suggest removing this advice and replacing it with the cloning instructions. This might also be an opportunity for mentioning updating `sys.path` strings to the location of the cloned dea-notebooks - unless that's redundant now with the updated python environment (i.e., dea-tools is installed)?

Reply:
It used to add a shortcut and icon for quickly launching DEA from the VDI program menu. I'm not sure what it does not - will check

Reply:
At the moment it does this:
``` 
cp /g/data/v10/public/digitalearthau/dea.desktop ~/.local/share/applications/
git clone https://github.com/GeoscienceAustralia/dea-notebooks.git --branch stable --single-branch --depth 1 ~/dea-notebooks/
```

Which is meant to copy a DEA icon into the applications menu for easy access (without having to launch notebooks from the terminal). Unfortunately it doesn't currently work, but I think I'd like to keep the option of fixing it in the future.

For now there is a new note down the bottom of the page which mentions that users should clone DEA Notebooks to a new location, and points to our wiki pages for doing that.